src/sentry/conf/server.py

```diff
@@ -684,6 +684,11 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue(
         "events.symbolicate_event_low_priority", routing_key="events.symbolicate_event_low_priority"
     ),
+    Queue("events.symbolicate_js_event", routing_key="events.symbolicate_js_event"),
```

Comment:
This can be removed now; Added in https://github.com/getsentry/sentry/pull/46936